software-engineering-policies/SourceControl/SourceControlPolicy.md

```diff
@@ -1,6 +1,6 @@
 # Source Control
 
-* All code **must** be stored in a source code repository.
+* All UKHO code **must** be stored in a source code repository.
```

Comment:
I gues this implicity applies to software teams only?

Reply:
Given the remit of the policy i guess yes. How we should be thinking and operating on the whole though we should have any UKHO code is a source control system for multiple reasons.